Source/cmake/OptionsGTK.cmake

```diff
@@ -343,28 +340,21 @@ endif ()
 SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER TRUE)
 
 if (USE_OPENGL_OR_ES)
-    # USE_OPENGL is the opposite of ENABLE_GLES2.
+    SET_AND_EXPOSE_TO_BUILD(USE_ANGLE ${ENABLE_WEBGL})
+    SET_AND_EXPOSE_TO_BUILD(USE_COORDINATED_GRAPHICS ON)
+    SET_AND_EXPOSE_TO_BUILD(USE_NICOSIA ON)
+    SET_AND_EXPOSE_TO_BUILD(USE_TEXTURE_MAPPER_GL ON)
+    SET_AND_EXPOSE_TO_BUILD(USE_EGL ON)
+    SET_AND_EXPOSE_TO_BUILD(USE_OPENGL ON)
+    SET_AND_EXPOSE_TO_BUILD(USE_OPENGL_ES ON)
+    SET_AND_EXPOSE_TO_BUILD(HAVE_OPENGL_ES_3 ON)
```

Comment:
Why is it OK to hardcode `HAVE_OPENGL_ES_3`?

Reply:
Because it's now a runtime dependency, and libepoxy supports it. In any case, it's actually unused, so we can probably just remove it in a follow up.

Reply:
Oh yeah, it is indeed unused. So please do remove it. (I don't care whether you do it here or in a follow-up.)